composer/datasets/glue.py

```diff
@@ -1,5 +1,18 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""`General Language Understanding Evaluation (GLUE; Wang et al, 2019)
```

Comment:
This doesn't render well in the summary table. We can reformat this as follows to match with other dataset docstrings. 

```
"""GLUE (General Language Understanding Evaluation) dataset. 

...

Please refer to the `GLUE benchmark <https://gluebenchmark.com/>`_ for more details. 


"""
```